Skuid_Techniques/ReferenceSelection/ReferenceSelectionPopup.xml

```diff
@@ -0,0 +1,240 @@
+<skuid__page unsavedchangeswarning="yes" personalizationmode="server" showsidebar="true" showheader="true">
+	<models>
+		<model id="NewTask" limit="1" query="false" datasource="salesforce" sobject="Task" createrowifnonefound="true">
+			<fields>
+				<field id="RecordTypeId"/>
+				<field id="RecordType.Name"/>
```

Comment:
Let's remove these so the page doesn't explode for users without recordtype enabled.
```suggestion
```